packages/lodestar/src/network/gossip/scoreMetrics.ts

```diff
@@ -0,0 +1,196 @@
+import {PeerScoreParams} from "libp2p-gossipsub/src/score";
+import {PeerStats} from "libp2p-gossipsub/src/score/peer-stats";
+import {MapDef} from "../../util/map";
+import {GossipType} from "./interface";
+import {GossipTopicCache} from "./topic";
+
+export type TopicScoreWeights<T> = {p1w: T; p2w: T; p3w: T; p3bw: T; p4w: T};
+export type ScoreWeights<T> = {
+  byTopic: MapDef<GossipType, TopicScoreWeights<T>>;
+  p5w: T;
+  p6w: T;
+  p7w: T;
+  score: T;
+};
+
+const gossipTypes = Object.values(GossipType);
+
+export function computeScoreWeights(
```

Comment:
`computeScores()` used to be a performance issue in gossipsub and we have to implement cache strategy there, I wonder if this affects lodestar's performance

Reply:
The only deployment I did didn't show to increase metrics scrape time. This function here is run once per peer and scrape peer. While in gossipsub is run for every message, which we receive hundreds per second.